step-templates/azure-check-vmss-provision-status.json

```diff
@@ -0,0 +1,138 @@
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Sensitive"
+        }
+      },
+      {
+        "Id": "6e0f8635-70ba-4b8a-b5b4-909dba3f9fd8",
+        "Name": "VMSS.Timeout.Value",
+        "Label": "Timeout (In Minutes)",
+        "HelpText": "**Required**\n\nHow long this step will wait (in minutes) for the VMSS and VMs in the VMSS to finish being created.",
+        "DefaultValue": "30",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "0392f153-73b0-44d4-9181-4f366c9012f7",
+        "Name": "VMSS.Timeout.ErrorHandle",
+        "Label": "Timeout Error",
+        "HelpText": "**Required**\n\nWhat will happen when a timeout occurs.\n\n- `Proceed`: The script will reconcile the VMs it can and then finish.  No error thrown.  \n- `Error`: Will throw an error and it will stop the deployment from proceeding.\n\nThe default is `Proceed`",
+        "DefaultValue": "Proceed",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Select",
+          "Octopus.SelectOptions": "Proceed|Proceed\nCancel|Cancel"
+        }
+      },
+      {
+        "Id": "80ef4fb9-c773-4d0a-82af-efdfb302e58f",
+        "Name": "VMSS.Duplicate.TimeInMinutes",
+        "Label": "Duplicate Run Time Allowance (in Minutes)",
+        "HelpText": "**Required**\n\nThis step is designed to wait for VMSS to finish scaling out.  However, Octopus isn't aware of VMSS and may attempt to run the same deployment multiple times as new targets are \"discovered.\"\n\nTypically, when this happens, a deployment is queued within a few minutes of the previous one finishing.  When that happens this step will treat that as a duplicate run.  \n\nThis setting indicates the number of minutes that must pass before a duplicate run is found.  The default is `3` minutes.",
+        "DefaultValue": "3",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "0eb3bed8-9259-440e-bff0-cb86af7da52e",
+        "Name": "VMSS.Duplicate.Handle",
+        "Label": "Duplicate Run Adjustment",
+        "HelpText": "**Required**\n\nWhat the step will do when a duplicate run is found.  The two options are:\n\n- `Cancel`: cancel the current runbook run or deployment.\n- `Proceed`: continue on with the current runbook run or deployment.\n\nThe default is `Proceed`.",
+        "DefaultValue": "Proceed",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Select",
+          "Octopus.SelectOptions": "Cancel|Cancel\nProceed|Proceed"
+        }
+      },
+      {
+        "Id": "adaf6a8c-453f-4b69-9ba6-f6a54b4f5bb9",
+        "Name": "VMSS.OldServers.ExcludeFromOutput",
```

Comment:
Not cause for any kind of concern, but I noticed the parameter: `VMSS.OldServers.ExcludeFromOutput` / `$excludeOldServers` value isn't written out like the other values in the script e.g.

`Write-Host "Exclude from output: $excludeOldServers"`

Reply:
Fixed